### DISCUSSION TITLE
Weekly release: 0.19.0.dev2025032500

### DISCUSSION THREAD

**Discussion by kaiyux** [NVIDIA/TensorRT-LLM#3099] [Announcements]
Hi,

The TensorRT-LLM team is pleased to announce that we have updated a weekly release `0.19.0.dev2025032500`, and pushed an update to the [Triton backend](https://github.com/triton-inference-server/tensorrtllm_backend/tree/main) this March 26, 2025. 

Moreover, we are thrilled to announce that our development workflow has officially transitioned to GitHub, and we eagerly anticipate collaborating with the global developer community! We are looking forward to your great work of contribution, feedback, or discussions on your brilliant ideas.

The `0.19.0.dev2025032500` dev release includes:

* Model Support
  - Added Qwen2VL model support in LLM API for PyTorch workflow.
* Features
  - PyTorch workflow
    - Added Autotuner for custom-op-compatible tuning process.
      - Added a Python-based Autotuner core framework for kernel tuning on PyTorch workflow.
      - Applied the Autotuner to fused MoE and NVFP4 linear operators for concept and performance evaluations.
    - Added guided decoding support (XGrammar integration).
    - Added pipeline parallelism support for the overlap scheduler in `PyExecutor`.
  - TensorRT workflow
    - Added LoRA support for Phi-2 model.
    - Added vision encoders with tensor parallelism and context parallelism support. Refer to `examples/vit/README.md`.
  - Added returning context logits support for `trtllm-serve`.
  - [AutoDeploy] Added expert parallelism (EP) support for MoE models.

The cut-off commit to this release is c2ffce7dbd283000255948ae04792f13f13e05dc.

Thanks,
The TensorRT-LLM Engineering Team
